tests/common/plugins/loganalyzer/__init__.py

```diff
@@ -15,6 +15,8 @@ def loganalyzer(duthost, request):
         logging.info("Log analyzer is disabled")
         yield
         return
+    # Force rotate logs
+    duthost.shell("/usr/sbin/logrotate /etc/logrotate.conf > /dev/null 2>&1")
```

Comment:
Missing `-f` option?

```
admin@str-msn2700-02:~$ /usr/sbin/logrotate --help
Usage: logrotate [OPTION...] <configfile>
  -d, --debug               Don't do anything, just test (implies -v)
  -f, --force               Force file rotation
  -m, --mail=command        Command to send mail (instead of `/usr/bin/mail')
  -s, --state=statefile     Path of state file
  -v, --verbose             Display messages during rotation
  -l, --log=STRING          Log file or 'syslog' to log to syslog
      --version             Display version information

Help options:
  -?, --help                Show this help message
      --usage               Display brief usage message
```

Reply:
updated, thanks.